pkg/gadgets/run/tracer/tracer.go

```diff
@@ -130,6 +137,11 @@ func (t *Tracer) Init(gadgetCtx gadgets.GadgetContext) error {
 		}
 	}
 
+	// TODO: how to check that other parameters like containername, namespace, etc are not set?
+	if ifaceParam := params.Get(types.IfaceParam); ifaceParam != nil {
+		t.ifaceName = ifaceParam.AsString()
+	}
```

Comment:
nit: It doesn't need to be done for any kind of programs but only for `SchedCLS`.

Reply:
You said that, they don't have the variable so it should be empty for that case.